From 01627fd3ae799d800769084dcae62c5b783fd9d9 Mon Sep 17 00:00:00 2001
From: Nguyễn Long <hoanglong.lumi404@gmail.com>
Date: Thu, 13 Apr 2023 05:51:59 +0000
Subject: [PATCH] zygote: Add ged to whitelisted paths

Signed-off-by: Nguyễn Long <hoanglong.lumi404@gmail.com>
---
 core/jni/fd_utils.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/jni/fd_utils.cpp b/core/jni/fd_utils.cpp
index 8ba2559e..31b05a12 100644
--- a/core/jni/fd_utils.cpp
+++ b/core/jni/fd_utils.cpp
@@ -48,6 +48,7 @@ static const char* kPathWhitelist[] = {
   "/dev/urandom",
   "/dev/ion",
   "/dev/dri/renderD129", // Fixes b/31172436
+  "/proc/ged",
 };
 
 static const char kFdPath[] = "/proc/self/fd";
-- 
2.40.0
